packaging/installer/README.md

```diff
@@ -333,7 +333,7 @@ This is how to install the latest Netdata version from sources on FreeBSD:
 
 ```sh
 # install required packages
-pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof
+pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy liblz4 libuv json-c
```

Comment:
I don't have clean system to test this on, but have you got openssl-devel installed as well?

Reply:
Everytime that I installed FreeBSD I had the openssl installed, this is the motive I did not add it in the list, did you already see a FreeBSD without it?

Reply:
OpenSSL is in base so there's no need to link it against ports unless you need bleeding edge for some reason

Reply:
The version in base doesn't include pkg-config info and we try to autodetect the package at install time. I think it works fine in the regular installer, it is only the CMake routines that give a problem at the moment, but I thought I would check.

Reply:
The installer should find the libcrypto.so from base, but there is no pkg-config to work against in that case. I'll install pfSense tomorrow morning and check the initial state and the fix.

Reply:
On BSD I agree 100%, but the problem with pfSense is a little more complicated, it looks like that pfSense is not keeping the same soft link used on FreeBSD and we are installing package from FreeBSD repository, this is the motive our user reported the issue, I think the best way to solve this is either to compile the packages with the correct libraries or to link static with json-c, considering that we began to have this problem between both Operate System.

Reply:
When I test this on a fresh 2.4.4-RELEASE both json-c and liblz4 are installed by default. Is this a packaging problem on the `-p3` release candidate that disappears if you do an upgrade to the released version?

Reply:
I did not test on -p3, but this list of packages is for FreeBSD, this PR fixes both, because I saw neither FreeBSD nor pfSense specification would allow the users to have dbengine activated.